Modern/wrapperPlugins/src/test/java/org/bonej/wrapperPlugins/IntertrabecularAngleWrapperTest.java

```diff
@@ -84,7 +85,7 @@ public void tearDown() {
 	@Test
 	public void testAngleResults() throws Exception {
 		// SETUP
-		final Predicate<String> nonEmpty = s -> !s.equals(SharedTable.EMPTY_CELL);
+		final Predicate<Double> nonEmpty = Objects::nonNull;
```

Comment:
This custom `Predicate` is now redundant, because it adds nothing new to `Objects::nonNull`